docs/test-v1/smoke-test-2.md

```
@@ -0,0 +1,128 @@
+| Status | Count |
+|--------|-------|
+| ✅ Fixed from Round 1 | 29 |
+| 🔴 Still broken | 2 |
+| 🆕 New issues found | 4 |
+
+---
+
+## ✅ Fixed Since V1 (29)
+
+### 🌐 General / All Pages
+| V1 # | Issue | Type | Device | GitHub Issue |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)